spacelift/data_stack.go

```diff
@@ -219,10 +234,15 @@ func dataStack() *schema.Resource {
 				Computed:    true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Description: "ID of the GitHub Enterprise application",
```

Comment:
I think it should be "ID of the GitHub Enterprise integration".

Reply:
true